File: src/main/java/com/p2p/lending/service/DopeService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Dope;

public interface DopeService {
	int insert(Dope dope);
	//分页查询
	public List<Dope> findDope(Map<String, Object> map);
	//查询总行数
	public List total();
	//删除
	public void batchDeletes(Integer did);
}

File: src/main/java/com/p2p/lending/service/DopeService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Dope
import org.springframework.stereotype.Service

@Service
interface DopeService {
    fun insert(dope: Dope): Int
    //分页查询
    fun findDope(map: Map<String, Any>): List<Dope>

    //查询总行数
    fun total(): List<*>

    //删除
    fun batchDeletes(did: Int?)
}
